js_modules/dagster-ui/packages/ui-core/src/app/FeatureFlags.oss.tsx

```diff
@@ -4,7 +4,7 @@ export enum FeatureFlag {
   flagSidebarResources = 'flagSidebarResources',
   flagDisableAutoLoadDefaults = 'flagDisableAutoLoadDefaults',
   flagLegacyRunsPage = 'flagLegacyRunsPage',
-  flagSelectionSyntax = 'flagSelectionSyntax',
+  flagSelectionSyntax = 'flagSelectionSyntax-always-on',
```

Comment:
renaming it so that if anyone toggled it off its now back to the default of on with no way to disable it after this PR. From looking at datadog nobody in cloud has turned it off.